makelib/golang.mk

```diff
@@ -100,6 +99,15 @@ else
 GOFMT := $(TOOLS_HOST_DIR)/gofmt$(GOFMT_VERSION)
 endif
 
+# We use a consistent version of golangci-lint to ensure everyone gets the same
+# linters.
+GOLANGCILINT_VERSION := 1.15.0
+ifneq ($(findstring $(GOLANGCILINT_VERSION),$(GO_VERSION)),)
```

Comment:
why are we comparing `GOLANGCILINT_VERSION` to `GO_VERSION`?

Reply:
Because this was copied and pasted from the gofmt logic while I was half awake, haha. Will fix.